src/components/cards/ListViewTaskCard.tsx

```diff
@@ -43,15 +43,14 @@ export const ListViewTaskCard = ({
         className="task-list-card"
         sx={{
           userSelect: 'none',
-          ':hover': {
-            bgcolor: (theme) => theme.color.gray[100],
-          },
+          cursor: 'default',
+          borderBottom: (theme) => `1px solid ${theme.color.borders.borderDisabled}`,
```

Comment:
hover bgColor is missing

Reply:
![image](https://github.com/copilot-platforms/tasks-app/assets/46821825/16cfb5ea-5ba4-4a9a-91c2-bb313e83bbb3)

hover state is not shown in figma design.

Reply:
Hover color has been changed I guess but it is there. Please have a look at the image below - 

![Screenshot from 2024-07-10 18-46-54](https://github.com/copilot-platforms/tasks-app/assets/38468429/c818002f-1199-455e-ac55-cf879eb35dc2)